tests/inference/batch_tests.json

```diff
@@ -0,0 +1,1136 @@
+[
+    {
+        "description": "YOLOv5n Object Detection",
+        "type": "object_detection",
+        "image_url": "https://source.roboflow.com/ACrZ7Hz8DRUB1NBMMtDoQK84Hf22/z5Kg76baWgoZXQPnNmry/original.jpg",
```

Comment:
Should we mark these as "do not delete" somehow (or perhaps rehost on `media.roboflow.com` in a `test-images` directory) in so that we don't someday delete the project in the app and mess up our tests?

Reply:
Good catch, probably should

(They are in my project that I use for testing everything so I will not delete it but you are right that we should probably rehost. Gonna punt on this for now in the interest of making progress on other stuff)